README.md

```diff
@@ -20,12 +20,49 @@ Any exceptions to this are noted in the individual `README.md` files.
 
 ## Installation
 
-TBD
+The repository layout structure is as defined by [the specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md). The `api`
+directory includes header-only abstract classes and no-op implementations that comprise the OpenTelemetry API following
+[the
+specification](https://github.com/open-telemetry/opentelemetry-specification).
+The `sdk` folder is the reference implementation of the API.
+
+The library is distributed in source-only format. No shared/static library or distribution packages (rpm, deb, etc.) are made available as part of Release. See [RELEASING.md](./RELEASING.md).
+
+This project supports `Bazel` and `CMake` builds.
+
+### Using CMake
+On most platforms with all dependencies installed, the following command will compile the source files, and install the header files:
+
+```sh
+cmake -Bcmake-out
+cmake --build cmake-out
+sudo cmake --build cmake-out --target install
+
+#To run the tests
+cd cmake-out && ctest
+```
 
-## Quick Start
+In case the library is distributed through package manager, below is the recommended deployment directory structure for header and librarie(s) on Linux and MacOS platforms.
 
+```
+/usr/local/lib:  For shared or static opentelemetry-cpp library
+/usr/local/include/opentelemetry: For api headers
+/usr/local/include/opentelemetry/sdk: For sdk headers
+/usr/local/src: For installing source package ( say libopentelemetry-dev )
+
+```
+
+For more details on configuration and installation, please refer to [INSTALL.md](./INSTALL.md).
+
+
+### Using Bazel
 TBD
```

Comment:
Instead of TBD, can we leave this section out and create an issue to add it?

Reply:
I think we should just skip the install entirely and handle it outside of README, splitting it maybe in separate docs:
- build with bazel
- build with cmake
- build with vcpkg (in essence, package manager helper around cmake)
- build with Visual Studio 2019 (CMake, GUI)
- easy-to-use one-liner build scripts in tools ( `build.sh` and `builld.cmd` for those who don't want to read the docs)

Reply:
I'm actually working right now to do these write-ups..

Reply:
Have removed the TBD.